curbs/README.md

```diff
@@ -263,6 +308,7 @@ A Curb Zone is represented as a JSON object, whose fields are as follows:
 | `entire_roadway`| Boolean | Optional | If "true", this curb location takes up the entire width of the roadway (which may be impassible for through traffic when the Curb Zone is being used for parking or loading). This is a common condition for alleyways. If `entire_roadway` is `true`, `street_side` MUST NOT be present. |
 | `curb_area_ids`| Array of [UUID][uuid] | Optional | The ID(s) of the [Curb Areas](#curb-area) that this Curb Zone is a part of. If specified, the areas identified MUST be retrievable through the Curb API and its geographical area MUST contain that of the Curb Zone. |
 | `curb_space_ids`| Array of [UUID][uuid] | Optional | The ID(s) of the [Curb Spaces](#curb-space) that this Curb Zone contains. If specified, the spaces identified MUST be retrievable through the Curb API and its geographical area MUST be contained in this Curb Zone. |
+| `curb_object_ids` | Array of [UUID][uuid] | Optional | The ID(s) of the [Curb Objects](#curb-object) that this Curb Zone is related to. If specified, the objects identified MUST be retrievable through the Curb API. Curb Objects can be related to a Curb Space or a Curb Zone. |
```

Comment:
This description should specify that the curb objects listed are the objects that are in its 'area of influence'. The Objects that relate to this curb zone. So a locker's ID might be listed in 4 curb zones along a block face. Or a camera's ID might be listed in 3 curb zones that the camera has visibility into. Or the parking meter might be useable by 5 zones on a block.

Reply:
Added text for this in the latest commit. I am still a little weary of how this can be interpreted (e.g. a camera is located in one curb zone, but monitors several curb zones, how will a user know which is which). But I think with the lat/long available for an object this should be covered (e.g. you use the lat/long to specify where the curb object is, then associate it with any zones it covers). Another example: a pay station's lat/long shows where it is physically, and this field lets you indicate all the curb zones it covers.

Reply:
Resolving so we can review the latest language in a new comment